### DISCUSSION TITLE
node

### DISCUSSION THREAD

**Discussion by mmmmmxx** [linguanyuan/server#1] [Q&A] [answered]
How do I catch error events for streams?

**Comment by linguanyuan** [comment, marked as answer]
Listen for error events, similar to the way you subscribe to EventEmitter's error events.

**Comment by mmmmmxx** [comment]
流有哪些典型应用？

**Reply by linguanyuan** [reply]
流在文件读写、网络请求、数据转换、音频、视频等方面有很广泛的应用。
